src/pages/organizers/create/OrganizerForm.tsx

```diff
@@ -55,10 +64,28 @@ const typeAndThemeStepConfiguration: StepsConfiguration<'nameAndUrl'> = {
   }),
 };
 
-const configurations = [typeAndThemeStepConfiguration];
+const configurations = [
+  typeAndThemeStepConfiguration,
+  {
+    ...additionalInformationStepConfiguration,
+    shouldShowStep: () => true,
+    variant: AdditionalInformationStepVariant.ORGANIZER,
+    scope: 'organizers',
+  },
+];
 
-const OrganizerForm = () => {
+const OrganizerForm = (props) => {
+  const { t } = useTranslation();
   const { form } = useParseStepConfiguration(configurations);
+  const { query, push, pathname, reload } = useRouter();
+  const organizerId = query?.organizerId;
+
+  const scope = 'organizers';
+  const createOrganizer = useCreateOrganizerMutation({
+    onSuccess: () => push(`/${scope}/${organizerId}/preview`),
```

Comment:
Should probably redirect to edit?
```suggestion
    onSuccess: () => push(`/${scope}/${organizerId}/edit`),
```